### ISSUE TITLE
Swagger OpenAPI Support?

### ISSUE THREAD

**Issue by escanaan** [HewlettPackard/python-ilorest-library#71]
Is HP working on a swagger.yml file? I need to run some mocking scenarios...found some tools from redfish repositories but they don't work as I expected.

https://github.com/DMTF/Redfish-Mockup-Creator
https://github.com/DMTF/Redfish-Mockup-Server

I also found from this statement from DMTF where they say OpenAPI support still work-in-progress as of 2018Q3:

https://www.dmtf.org/sites/default/files/Redfish_OpenAPI_Support_2018Q3%28WIP%29.pdf

Are you @Yergidy using this to generate your libraries?

Thank you.

**Comment by rajeevkallur** [comment]
No, we did not use any yml to generate library. But iLO team would have used some method to create redfish server.
Here is something I found which could be helpful.
We can use Swagger Editor to create your OAS definition and then use Swagger Codegen to generate server implementation.
We can use the Swagger UI to visualize and document your OAS definition.
Design, document and develop APIs as a team using SwaggerHub.

**Comment by alxvls** [comment]
This is such a weird conversation :) it would be really helpful if HPE uploaded OpenAPI spec used for redfish service somewhere
